LayoutTests/fast/images/animated-image-mp4-crash.html

```diff
@@ -2,10 +2,13 @@
 <html>
 <body>
     <img>
-    <script src="../../resources/js-test-pre.js"></script>
+    <script src="../../resources/js-test.js"></script>
     <script>
         window.jsTestIsAsync = true;
 
+        if (window.internals)
+            internals.clearMemoryCache();
```

Comment:
I don't really understand why this resolves the flakiness. Does `image.decode()` not resolve its promise if the image exists in the MemoryCache and we've tried to (and failed to) decode it before? I wonder if that's actually what the spec wants. Either way, can you please explain a little in a comment here to make it clear why we're doing this.